shared/constants.js

```diff
@@ -60,6 +60,16 @@ const GET_META_REGEXP = /\(meta\(\)\.(.*?)\)/;
 const GET_NODES_REGEXP = /"nodes":(\[.*?\])/;
 const GET_PARTITION_HASH_REGEXP = /(HASH|hash)\((.*?)\)$/;
 
+/**
+ * @enum {string}
+ */
+const PK_SEGMENT_TYPE = {
+	constant: 'constant',
```

Comment:
Shouldn't enum options names be in upper case?

Reply:
No. Please read https://app.gitbook.com/o/HBtg1gLTy0nw4NaX0MaV/s/RsuO9R6FbfIJtofdqr4L/guides-and-how-to/guidelines-for-code-consistency